### PR TITLE
private testnet setup

### PR DESCRIPTION
**Pull request by cedar-book**
updated new private testnet setup page info

### CODE REVIEW

**Comment by cedar-book** [comment]
Hi @pmconrad, I updated Set up Private testnet page and related page. If you have chance, please review them?

File: docs/bts_guide/tutorials/how_to_get_key_pairs.rst [hunk 1]
```diff
@@ -40,11 +40,11 @@ Use this option for production.
...
 
 
-get_dev_key option
+get_dev_key 
 ----------------------------------------------
 
-Use this option for testing purpose. 
-This ``get_dev_key`` option can generate more than one key pairs at the same time in one command line. The option is simply combine the prefix with all given suffixes, compute a hash over each combination, and use that hash as a private key, from which the matching public key and address are generated.
+Use this function for testing purpose. 
+This ``get_dev_key`` function can generate more than one key pairs at the same time in one command line. The function is simply combine the prefix with all given suffixes, compute a hash over each combination, and use that hash as a private key, from which the matching public key and address are generated.
```
Comment (abitmore) [inline docs/bts_guide/tutorials/how_to_get_key_pairs.rst]: `get_dev_key --help` now produces better output. Please update this document. Thanks.

File: docs/development/testnets/private_testnet-v2.rst [hunk 2]
```diff
@@ -554,11 +559,14 @@ This section includes instructions for the following:
 * Setting a password and Unlock the cli wallet  
 
 
-.. Important:: Each wallet is specifically associated with a single chain, specified by its chain ID. This is to protect the user from (e.g., unintentionally) using a testnet wallet on the real chain. The Chain ID passed to the Cli_Wallet needs to match the Chain ID generated and used by the witness node.
+.. Important:: Each wallet ``wallet.json`` is specifically associated with a single chain, specified by its chain ID. This is to protect the user from (e.g., unintentionally) using a testnet wallet on the real chain. The Chain ID passed to the cli_wallet needs to match the Chain ID generated and used by the witness node.
```
Comment (abitmore) [inline docs/development/testnets/private_testnet-v2.rst]: The default wallet file name is `wallet.json`, but the user use other file names as well.

File: docs/development/testnets/private_testnet-v2.rst [hunk 3]
```diff
@@ -650,13 +658,17 @@ To process this step, we prepared initial accounts and initial balances informat
...
   
-	//For example: 
-	import_key init30-test "5JU3yZnDy5Gf9gS4iQwSS1zDLzP3ECmRfWv6kx76WxnufTQRAqr"
+	# For example: 
+	import_key init30-test "5JG5thpLiuTG1ANiV9j4EyDHCXjvM67NRLtYSRGhusL5wg9CahY" # active_key
+	
+	import_key init30-test "5JU3yZnDy5Gf9gS4iQwSS1zDLzP3ECmRfWv6kx76WxnufTQRAqr"  # owner_key
 
+	
+.. Note:: The active key is equal to the memo key, which is required for transfers with a memo.
```
Comment (abitmore) [inline docs/development/testnets/private_testnet-v2.rst]: > active key is equal to the memo key

They can be different, although they're the same in this example.

File: docs/development/testnets/private_testnet-v2.rst [hunk 4]
```diff
@@ -962,9 +1002,13 @@ If you want to set up a second node (with the same genesis file) and connect it
 		
 
 	
+.. important:: 
 
-	
+  - The config files should **not** contain the same witness ids and private keys. Otherwise, both nodes would produce blocks simultaneously, which means they would end up on different chain forks.
+  - Each node should use only a subset of the witnesses, so block production alternates between them.
```
Comment (abitmore) [inline docs/development/testnets/private_testnet-v2.rst]: Can use same witness IDs but different keys in different nodes, which can be used to "hot-switch" among production nodes.

File: docs/development/testnets/private_testnet_genesis_example.rst [hunk 5]
```diff
@@ -15,18 +15,40 @@ For the use of private testnet, you must put an account into the genesis file wi
...
+
+- For the private testnet, each public key (key pairs) should be generated. You use different keys to set up the private testnet genesis file ``active_key`` and ``owner_key`` for each **initX** account. You can use your own "name" settings.
+
+- "initial_accounts": The names "initx"..."initxx" can be chosen freely. **Note::** the "initial_witness_candidates"- ``owner_name`` and "initial_committee_candidates"- ``owner_name`` refer the the "initial_account" by ``name``.  
+
+- ``name``: There is no direct connection between the account ``name`` and keys, so you can change the name and insert keys from ``get_dev_key`` results. If you import the corresponding private keys into cli_wallet, then cli_wallet will recognize the accounts as its own.
+
+- The UI generates keys by hashing name, password and a key type suffix. Therefore, only if you want to use the UI, there is a connection between name, password and keys.
+
+- "max_care_supply" refers to **TEST** in testnet, and to **BTS** in mainnet.
```
Comment (abitmore) [inline docs/development/testnets/private_testnet_genesis_example.rst]: `core` but not `care`.

File: docs/development/testnets/private_testnet_genesis_example.rst [hunk 6]
```diff
@@ -321,7 +343,7 @@ Sample private testnet genesis file
 	  ],
 	  "initial_assets": [],
 	  "initial_balances": [{
-		  "owner": "--- set a public key ---",
+		  "owner": "--- set an address key ---",
```
Comment (abitmore) [inline docs/development/testnets/private_testnet_genesis_example.rst]: `address` but not `address key`.

**Pull request merged**